test_pull_requests

```diff
@@ -1074,6 +1107,7 @@ popd
       comments = comments ? comments : get_comments(issue_id, repo)
 
       comments.each do |comment|
+        $stderr.puts "get_comment_matching_regex, comment['body']: #{comment['body']}, comment['user']['login']: #{comment['user']['login']}"
```

Comment:
I think all of these debug statements should be evaluated -- should we have logging here forever?